electron/css/about.css

```diff
@@ -31,11 +32,9 @@ body {
   display: flex;
   justify-content: center;
   align-items: center;
-  font-family: -apple-system, ".SFNSText-Regular", "Helvetica Neue", "Segoe UI", Arial, sans-serif;
+  font-family: -apple-system, '.SFNSText-Regular', 'Helvetica Neue', 'Segoe UI', Arial, sans-serif;
   font-size: 11px;
   text-align: center;
-  user-select: none;
```

Comment:
Should we make everything user selectable or just the version information?

Reply:
Don't see any special reason to have only version if we are about to make it selectable.. :)

We could also simply copy that info with Ctrl/Cmd+C in that window.. like any prompt dialog boxes on Windows!! (That thing I miss on macOS)